NineChronicles.Headless/GraphTypes/AddressQuery.cs

```diff
@@ -69,6 +72,36 @@ public AddressQuery()
                     }
                 ),
                 resolve: context => Addresses.GetRaiderListAddress(context.GetArgument<int>("raidId")));
+
+            Field<ListGraphType<AddressType>>(
```

Comment:
Since it expects `null` and `[]`, or `["0x..."]`, not `[null]`, it should be like the below line.

```suggestion
            Field<ListGraphType<NonNullableGraphType<AddressType>>>(
```